src/wallet/rpcdump.cpp

```diff
@@ -88,6 +88,38 @@ bool GetWalletAddressesForKey(CWallet * const pwallet, const CKeyID &keyid, std:
     return fLabelFound;
 }
 
+bool ImportPrivateKey(CWallet * const pwallet, const UniValue& privkey, const std::string& label = "", const int64_t timestamp = 1)
```

Comment:
In commit "[wallet] [rpc] Move private key import functionality out of importprivkey",

`static bool ...`?